src/app/components/form/form.component.ts

```diff
@@ -221,7 +233,7 @@ export class FormComponent implements OnInit, OnChanges {
     }
   }
 
-  public runDomainCheck() {
+  public runDomainCheck(submitNotValid = false) {
```

Comment:
Maybe give the argument a positive name?

Reply:
I inverted the state so it has a positive name